bash/bash_profile_vs_bashrc.md

```diff
@@ -0,0 +1,40 @@
+# `.bash_profile` vs `.bashrc`
+
+`.bashrc` is sourced on every interactive shell start. Here are some ways to
+start `bash(1)` interactively:
+
+    bash
+    bash -i
+    bash -ic 'echo Hello!'
```

Comment:
Not sure it will improve anything, but triple-backtick code fence these lines for syntax highlighting?

    ```bash
    ... code
    ```